examples/run_glue.py

```diff
@@ -61,20 +61,7 @@ class ModelArguments:
     )
 
 
-def main():
-    # See all possible arguments in src/transformers/training_args.py
-    # or by passing the --help flag to this script.
-    # We now keep distinct sets of args, for a cleaner separation of concerns.
-
-    parser = HfArgumentParser((ModelArguments, DataTrainingArguments, TrainingArguments))
-
-    if len(sys.argv) == 2 and sys.argv[1].endswith(".json"):
-        # If we pass only one argument to the script and it's the path to a json file,
-        # let's parse it to get our arguments.
-        model_args, data_args, training_args = parser.parse_json_file(json_file=os.path.abspath(sys.argv[1]))
-    else:
-        model_args, data_args, training_args = parser.parse_args_into_dataclasses()
-
+def run_glue(model_args, data_args, training_args):
```

Comment:
I put `run_glue` here as a placeholder, but could be replaced by something better like `run()` or something else.

I chose to keep `main()` as the method that can launch the full training, as it is what we call from tests.